modules/sdk/node-build/esbuild/writeExportBridge.mjs

```diff
@@ -0,0 +1,45 @@
+import fs from 'fs/promises';
+import path from 'path';
+
+import getExportedSymbols from '../util/getExportedSymbols.mjs';
+import getExportBridgePath from './getExportBridgePath.mjs';
+
+export default async function writeExportBridge(moduleName) {
+	const exportBridgePath = getExportBridgePath(moduleName);
+
+	const symbols = getExportedSymbols(moduleName);
+
+	const regularSymbols = Object.keys(symbols)
+		.filter((symbol) => symbol !== 'default' && symbol !== '__esModule')
+		.join(',\n');
+
+	//
+	// If the exported object was generated by a harmony aware tool, we
```

Comment:
What is the specific tool? Just thinking so we can use it later as needed

Reply:
Any tool like webpack, babel, etc...